.github/workflows/release.yaml

```diff
@@ -1,23 +1,35 @@
 name: Release
 
 on:
-  workflow_dispatch:
-    inputs:
-      version:
-        description: 'The new version number. Example: 1.40.1'
-        required: true
+  push:
+    branches:
+      - master
 
 jobs:
-  release:
+  Release:
+    if: "${{ startsWith(github.event.head_commit.message, 'chore(release)') }}"
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
         with:
-          fetch-depth: 0 # gives standard-version access to all previous commits
-
-      - name: Update version number in files
-        run: scripts/prepare_release.sh "${{ github.event.inputs.version }}"
-
+          fetch-depth: 0
+  
+      - uses: actions/github-script@v5
+        with:
+          script: |
+            // Parse the new changes from the changelog
```

Comment:
would be good to have it as a local action and share between workflows since duplicated

Reply:
yeah good idea. i've put the script into a shared file.